src/library/scala/Option.scala

```diff
@@ -229,6 +315,13 @@ sealed abstract class Option[+A] extends Product with Serializable {
 
   /** Tests whether the option contains a given value as an element.
    *
+   * This is equivalent to:
+   * {{{
+   * option match {
+   *   case Some(elem) => true
```

Comment:
I believe this is only true if `elem` is a literal. Otherwise, it either needs to be in backticks? or use a guard (e.g. `case Some(x) if x == elem`)

Reply:
You're right, good catch.